todo_app/templates/index.html

```diff
@@ -11,8 +24,25 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items%}
+          <li>{{item.id}} {{item.title}}  <b>{{item.status}}</b></li>
+        {% endfor %}
       </ul>
+      <form action="/add-item" method="post">
+        <label for="field_name">Add Item:</label>
+        <input type="text" id="field_name" name="field_name">
+        <input type="submit" value="Submit">
+      </form>
+      <form action="/update-item" method="post">
+        <label for="id_to_edit">Mark as complete (Item ID):</label>
+        <input type="text" id="id_to_edit" name="id_to_edit">
+        <input type="submit" value="Submit">
+      </form>
```

Comment:
Minor, but I tend to prefer the user-experience of never having to type in an id if I can avoid it; for example here we could create a "Mark as complete" button for _each_ existing entry and set the id as a hidden input so that it gets passed through the same but the user never needs to type an id in - something like:
```html
{% for item in items%}
  <li>
    <form action="/update-item" method="post">
      {{item.id}} {{item.title}}  <b>{{item.status}}</b>
      <input type="text" id="id_to_edit" name="id_to_edit" value="{{item.id}}" hidden>
      <input type="submit" value="Mark as complete">
    </form>
  </li>
{% endfor %}
```